pkg/apis/componentconfig/types.go

```diff
@@ -120,10 +121,74 @@ const (
 	HairpinNone = "none"
 )
 
+type ExperimentalKubeletConfiguration struct {
```

Comment:
nit: Add a description stating that this is an internal unversioned object and why it exists.